Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM node:20-slim
+RUN apt-get update && apt-get -y install dumb-init gcc g++ make libsqlite3-dev zlib1g-dev
```

Comment:
can you convert this to multistage build? I think it will save us a fair bit on bandwidth and speed up pulling it locally. most of our bikehopper dockerfiles are multistage builds (e.g. [bikehopper-ui](https://github.com/bikehopper/bikehopper-ui/blob/main/Dockerfile)). i can help you with it if you want, or just do it myself.